docs/docsite/rst/network/user_guide/faq.rst

```diff
@@ -0,0 +1,34 @@
+.. _network_faq:
+
+*******************
+Ansible Network FAQ
+*******************
+
+How can I improve performance for network playbooks?
+====================================================
+
+* Set ``strategy: free`` at the playbook level if you are running on multiple hosts
+
+The ``strategy`` plugin controls the ordering of tasks on hosts. When ``strategy=linear``, Ansible will wait until the current task has run on all hosts before starting the next host. When ``strategy=free``, Ansible will execute the next task on each host as soon as that host completes the current task, so overall execution is not waiting for the slowest host. 
```

Comment:
I was wondering if we should link to the `linear` and `free` pages though these seem low in detail

http://docs.ansible.com/ansible/latest/user_guide/playbooks_strategies.html
https://docs.ansible.com/ansible/devel/plugins/strategy/linear.html
https://docs.ansible.com/ansible/devel/plugins/strategy/free.html

I wonder if we should flesh those pages out a bit and include links from here. That would mean as the reference material is improved we guide people to those pages.